koku/api/report/aws/serializers.py

```diff
@@ -71,6 +60,15 @@ def __init__(self, *args, **kwargs):
             # Add AWS tag keys to allowable fields
             self.fields.update(tag_keys)
 
+        and_fields = {'and:' + field: StringOrListField(child=serializers.CharField(),
```

Comment:
This block for and_fields, or_fields, self.fields.update() seem to be the same for all `GroupbySerializer` and `FilterSerializer`  Would making an `api.serializers.py` to create a base class consolidate this?

Reply:
Agree with Doug. A common super-class probably makes sense here.